pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -209,6 +210,12 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 		}
 	}
 
+	// fixes https://issues.redhat.com/browse/OCPBUGS-5304
+	if reflect.DeepEqual(liveVotingMembers, healthyLiveVotingMembers) && len(healthyLiveVotingMembers) <= desiredControlPlaneReplicasCount {
+		klog.V(2).Infof("ignoring scaling down, all voting member are healthy, no unhealthy voting member found")
```

Comment:
Clarify the log message.
```suggestion
		klog.V(2).Infof("skip scale down: all voting members are healthy and membership does not exceed the desired cluster size of %v", desiredControlPlaneReplicasCount)
```
Also can you please confirm by looking at the operator's logs but I think this is the default case for a normal cluster and this would end up spamming the logs so V(2) may not be appropriate here.

Reply:
alright, shall i use `v(4)` ? @hasbro17